runtime/runtime_controller.h

```diff
@@ -119,7 +119,7 @@ class RuntimeController final : public WindowClient {
     std::string variant_code;
     std::vector<std::string> locale_data;
     std::string user_settings_data = "{}";
-    std::string lifecycle_state;
+    std::string lifecycle_state = "AppLifecycleState.detached";
```

Comment:
@xster This will be true as long as dart project is run after engine is initialized

Reply:
There is no direct api for engine to modify window.dart property. Even if we drill one down, it still have to go through shell, and that will be the same case as before.

Reply:
Got it. Thanks